change/@fluentui-react-progress-92befd38-f59c-467a-a4a4-3836c00c0c15.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

I think this needs to be minor to trigger the change to stable.

```suggestion
  "type": "minor",
```